http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/Boot.scala

```diff
@@ -585,6 +588,17 @@ object Boot extends IOApp {
         googleDependencies.googleResourceService
       )
 
+      val aksAlg = new AKSInterpreter[F](
```

Comment:
I added `AKSInterpreter` to `Boot.scala`, though it's not used for anything yet. :)

Eventually (in a separate PR) we'll update `AzurePubSubHandler` to handle createApp messages and invoke `AKSInterpreter`.